R/system.R

```diff
@@ -1482,3 +1482,14 @@ read_rds_file <- function(file, refhook = NULL){
     readRDS(file, refhook)
   }
 }
+
+#'Wrapper for readr::read_lines to support vector to data frame conversion
+#'@export
+read_raw_lines <- function(file, locale = readr::default_locale(), na = character(),
+                            skip = 0, n_max = Inf, progress = interactive()){
+  loadNamespace("readr")
+  line <- readr::read_lines(file, locale = locale, na = na, skip = skip, n_max = n_max, progress = progress)
+  df <- as.data.frame(line)
```

Comment:
The following should be easier to understand being explicit about column name instead of variable name implicitly becoming column name.
df <- as.data.frame(line=line)